contents/handbook/people/spending-money.md

```diff
@@ -42,21 +42,21 @@ Make sure you forward digital invoices to Brex from your PostHog email address -
 ### Revolut
 
 - Buy something on your Revolut card.
-- If it's a digital invoice, just forward it to ukinvoices@posthog.com. If it's a physical receipt, take a picture and forward it to the same place.
+- If it's a digital invoice, just forward it to ukinvoices@posthog.com with a note explaining what it is for. If it's a physical receipt, take a picture and forward it to the same place with a note. 
 - You need to submit receipts for _all_ purchases. 
 
 ### Your personal card
 
 - You shouldn't usually need to do this - if you can't use your PostHog card for some reason, ask Charles or Eltje to purchase it for you on theirs in the first instance. 
-- If you do need to use your personal card, just send your receipt to Charles directly and he'll process a manual payment. 
+- If you do need to use your personal card, just post the receipt in the #ops channel in Slack and tag Charles and Grace to get reimbursed manually.  
 
-Accidentally bought something on the company card when it was a personal expense? Don't worry! Again, just let Charles know _as soon as you become aware_ and he will provide you with the relevant bank details for you to repay the company. 
+Accidentally bought something on the company card when it was a personal expense? Don't worry! Again, just post in #ops _as soon as you become aware_ and we will provide you with the relevant bank details for you to repay the company. 
```

Comment:
I would also add a note that if you can request a refund directly (without hassle or time consumption) it might be easier for everyone. For instance, I've accidentally charged some personal Uber rides to my Brex card in the past and I just open the Uber app and change the payment method.